shell/persistent-screen.md

```diff
@@ -0,0 +1,33 @@
+## Persistent screen or tmux session on lxplus
+
+### Setting up password-less kerberos token
+
+In order for the kerberos token to be refreshed automatically, it must be possible to do so without a password.
+Therefore, we create a keytab (similar to a public ssh key) on lxplus:
+```bash
+# /usr/kerberos/sbin/ktutil
+ktutil:  add_entry -password -p USERNAME@CERN.CH -k 1 -e arcfour-hmac-md5
+Password for USERNAME@CERN.CH: 
+ktutil:  add_entry -password -p USERNAME@CERN.CH -k 1 -e aes256-cts
+Password for USERNAME@CERN.CH: 
+ktutil:  wkt USERNAME.keytab
+```
+### Making use of the keytab
+This will create a file called USERNAME.keytab in the current directory. It is strongly advised to but this into a part of your afs which only you (!!!) can access.
```

Comment:
I would add something along the lines of: "It is strongly recommended to store this file in a directory to which only you have access as anyone who obtains a copy of this file can use it to obtain tokens in your name."